misk-hibernate/src/main/kotlin/misk/hibernate/HibernateExceptionMappers.kt

```diff
@@ -74,7 +75,7 @@ internal class ResourceExhaustedExceptionMapper @Inject internal constructor() :
 
 internal object ConflictExceptionResponder {
   private val HEADERS: Headers =
-      Headers.of(listOf("Content-Type" to MediaTypes.TEXT_PLAIN_UTF8).toMap())
+      listOf("Content-Type" to MediaTypes.TEXT_PLAIN_UTF8).toMap().toHeaders()
```

Comment:
could follow-up to do just `headersOf("Content-Type", MediaTypes.TEXT_PLAIN_UTF8)`

Reply:
changed